packages/graphback/src/generators/resolvers/layered/LayeredRuntimeResolverGen.ts

```diff
@@ -0,0 +1,119 @@
+
+/**
+ * Generate runtime resolver layer using Apollo GraphQL format 
+ * and injected service layer. Service layer offers various capabilities like monitoring, cache etc. 
+ * so resolver logic can be kept simple and interchangable.
+ * 
+ * Resolvers are formatted using graphql-tools format
+ * 
+ * ```javascript
+ * const resolvers = {
+ *    Query: {...}
+ *    Mutation: {...}
+ *    Subscription: {...}
+ * }
+ * ```
+ * 
+ */
+export class LayeredRuntimeResolverGenerator {
+  private inputContext: InputModelTypeContext[]
+  private service: GraphbackCRUDService
+
+  constructor(inputContext: InputModelTypeContext[], service: GraphbackCRUDService) {
+    this.inputContext = inputContext
+
+    this.service = service;
+  }
+
+  public generate() {
+    const resolvers = {
+      Query: {},
+      Mutation: {},
+      Subscription: {}
+    };
+    for (const resolverElement of this.inputContext) {
+      if (resolverElement.config.disableGen) {
+        continue;
+      }
+
+      if (resolverElement.config.create) {
+        const resolverCreateField = getFieldName(resolverElement.name, ResolverType.CREATE);
+        // tslint:disable-next-line: no-any
+        resolvers.Mutation[resolverCreateField] = (parent: any, args: any, context: any) => {
+          return this.service.create(resolverElement, args.input, context)
+        }
+      }
+      if (resolverElement.config.update) {
+        const updateField = getFieldName(resolverElement.name, ResolverType.UPDATE);
+        // tslint:disable-next-line: no-any
+        resolvers.Mutation[updateField] = (parent: any, args: any, context: any) => {
+          return this.service.update(resolverElement, args.id, args.input, context)
+        }
+      }
+      if (resolverElement.config.delete) {
+        const deleteField = getFieldName(resolverElement.name, ResolverType.DELETE);
+        // tslint:disable-next-line: no-any
+        resolvers.Mutation[deleteField] = (parent: any, args: any, context: any) => {
+          return this.service.delete(resolverElement, args.id, context)
+        }
+      }
+      if (resolverElement.config.findAll) {
+        const findAllField = getFieldName(resolverElement.name, ResolverType.FIND_ALL, 's');
+        // tslint:disable-next-line: no-any
+        resolvers.Query[findAllField] = (parent: any, args: any, context: any) => {
+          return this.service.findAll(resolverElement, context)
+        }
+      }
+      if (resolverElement.config.find) {
+        const findField = getFieldName(resolverElement.name, ResolverType.FIND, 's');
+        // tslint:disable-next-line: no-any
+        resolvers.Query[findField] = (parent: any, args: any, context: any) => {
+          return this.service.findBy(resolverElement, args.fields, context)
+        }
+      }
+
+      this.createSubscriptions(resolverElement, resolvers)
+    }
+    // TODO relationships
+
+    return resolvers;
+  }
+
+  // tslint:disable-next-line: no-any
```

Comment:
hmm.. it is easier to disable this rule IMHO rather than letting all things to become any. 
Our usage of any is around generics. Usually people define single type like "Unknown" and then map that everywhere but I did not wanted to do this till we split to packages. 

No strong opinions